sopel_8ball/choices.py

```diff
@@ -118,3 +118,37 @@ def choices(self):
             "Would you take ‘no’ for an answer?",
             "Not in this lifetime!",
         )
+
+
+class Spooky(AbstractChoiceProvider):
+    """A spooky magic 8 ball."""
+    def choices(self) -> Tuple[str, ...]:
+        return (
+            # affirmative answers
+            "The ghost over your shoulder said yes.",
+            "The accursed screams in agreement.",
+            "The elders smile from the deep.",
+            "I've seen this in the dreams of a Great Old One.",
+            "For once eerie voices whisper in harmony.",
+
+            # non-committal answers
+            "The abyss remain silent.",
```

Comment:
Unless "the abyss" is a collective of beings? But I doubt that.
```suggestion
            "The abyss remains silent.",
```

Reply:
After some digging, it's clearly a very fr to en translation problem.